### DISCUSSION TITLE
Robot falls from auto generated world

### DISCUSSION THREAD

**Discussion by moritzboeker** [FieldRobotEvent/Virtual_Field_Robot_Event#25] [Q&A] [answered]
Dear FRE2021-team,

I have generated the example worlds for each task by using the bash scripts.
But unfortunately the robot is literally falling off that world and thus cannot be controlled. And there is a part of the ground that is lower than the rest of the field but the maize rows do not follow this shape but are set on ground of two heights.
Here is an image for clarification:
https://ibb.co/cF2FVQz

Do you have an idea how to come over this problem?

Kind regards,

Moritz

**Comment by moritzboeker** [comment, marked as answer]
Okay I just pulled the actual changes made from you some minutes ago and now the robot is there! Thank you

**Reply by ThijsRuigrok** [reply]
Happy to hear that you found the solution :)
